terraso_backend/apps/graphql/schema/story_maps.py

```diff
@@ -75,10 +93,29 @@ def resolve_configuration(self, info):
 
         return self.configuration
 
+    def resolve_membership_list(self, info):
+        user = info.context.user
+        if user.is_anonymous or self.membership_list is None:
+            return None
+
+        if self.created_by == user:
+            return self.membership_list
```

Comment:
question: Why treat this case differently than a normal member?

Reply:
You mean why don't we just create a new membership entry for the owner instead of handling the created by value?

Reply:
David, I think this is a similar conversation: https://github.com/techmatters/terraso-backend/pull/718#discussion_r1317455513